### PR TITLE
Update test_tender_data_limited

### PR DESCRIPTION
**Pull request by mykhaly**
Update test_tender_data_limited in order to add required fields in API v. 2.1 to negotiation and
negotiation.quick procurements
Add other not necessary field to test_tender_data_limited
Make test_tender_data_limited more randomized
Add `Відображення * ` test cases to limited proc.

---
This change is [Reviewable](https://reviewable.io/reviews/openprocurement/robot_tests/111)

### CODE REVIEW

**Comment by selurvedu** [comment]
Review status: 0 of 1 files reviewed at latest revision, 2 unresolved discussions.

---

*[op_robot_tests/tests_files/initial_data.py, line 196 \[r1\]](https://reviewable.io:443/reviews/openprocurement/robot_tests/111#-KDMB3GFUf6A5PSUwt7m:-KDMB3GGrUykNv2XtA3W:309003082) ([raw file](https://github.com/openprocurement/robot_tests/blob/d4aec7fac40fcb45f9fcddcc15628d958762ffc0/op_robot_tests/tests_files/initial_data.py#L196)):*
Що означають ці слова?

---

*[op_robot_tests/tests_files/initial_data.py, line 201 \[r1\]](https://reviewable.io:443/reviews/openprocurement/robot_tests/111#-KDMAuysQYkO5QoVhnXN:-KDMAuyuy1hv4QuR9btz:1512018690) ([raw file](https://github.com/openprocurement/robot_tests/blob/d4aec7fac40fcb45f9fcddcc15628d958762ffc0/op_robot_tests/tests_files/initial_data.py#L201)):*
Якщо тут передбачається лише один варіант, то для чого `random_element`?

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/openprocurement/robot_tests/111)*

**Comment by mykhaly** [comment]
Review status: 0 of 1 files reviewed at latest revision, 2 unresolved discussions.

---

*[op_robot_tests/tests_files/initial_data.py, line 196 \[r1\]](https://reviewable.io:443/reviews/openprocurement/robot_tests/111#-KDMB3GFUf6A5PSUwt7m:-KDNScRmluGNq_phlSd1:-1821425628) ([raw file](https://github.com/openprocurement/robot_tests/blob/d4aec7fac40fcb45f9fcddcc15628d958762ffc0/op_robot_tests/tests_files/initial_data.py#L196)):*
Можливі значення поля `cause` для `negotiation`.

---

*[op_robot_tests/tests_files/initial_data.py, line 201 \[r1\]](https://reviewable.io:443/reviews/openprocurement/robot_tests/111#-KDMAuysQYkO5QoVhnXN:-KDNSkTo9BcWoCST0X1D:2065630944) ([raw file](https://github.com/openprocurement/robot_tests/blob/d4aec7fac40fcb45f9fcddcc15628d958762ffc0/op_robot_tests/tests_files/initial_data.py#L201)):*
щоб було по аналогії з `negotiation`. Також маю думку, що можливо, в майбутньому, сюди теж докинуть пару нових значень і тоді треба буде лише дописати їх.

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/openprocurement/robot_tests/111)*

File: op_robot_tests/tests_files/limited_keywords.robot [hunk 1]
```diff
@@ -117,6 +117,23 @@ Resource           resource.robot
   ...      ${TENDER['TENDER_UAID']}
   ...      tenderID
 
+
+Відображення опису закупівлі
+  Звірити поле тендера  ${viewer}
+  ...      ${USERS.users['${tender_owner}'].initial_data}
+  ...      description
+
+Відображення причини вибору закупівлі
```
Comment (myroslav) [inline op_robot_tests/tests_files/limited_keywords.robot]: Відображення Підстави

File: op_robot_tests/tests_files/limited_keywords.robot [hunk 2]
```diff
@@ -117,6 +117,23 @@ Resource           resource.robot
...
+  Звірити поле тендера  ${viewer}
+  ...      ${USERS.users['${tender_owner}'].initial_data}
+  ...      causeDescription
+
+
+Відображення опису причини вибору закупівлі
```
Comment (myroslav) [inline op_robot_tests/tests_files/limited_keywords.robot]: Відображення Обгрунтування

File: op_robot_tests/tests_files/negotiation.quick.robot [hunk 3]
```diff
@@ -126,6 +126,27 @@ ${broker}       Quinta
   ...  ${USERS.users['${viewer}'].broker}
   Відображення ідентифікатора закупівлі
 
+
+Відображення опису переговорної процедури за нагальною потребою
```
Comment (myroslav) [inline op_robot_tests/tests_files/negotiation.quick.robot]: аналогічно вище

File: op_robot_tests/tests_files/limited_keywords.robot (hunk 2 above)
Comment (mykhaly) [inline op_robot_tests/tests_files/limited_keywords.robot]: Done.

File: op_robot_tests/tests_files/limited_keywords.robot (hunk 1 above)
Comment (mykhaly) [inline op_robot_tests/tests_files/limited_keywords.robot]: Done.

**Comment by myroslav** [comment]
:lgtm:

---

Reviewed 4 of 5 files at r4, 1 of 1 files at r5.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/openprocurement/robot_tests/111#-:-KDTu9-pXCkyjmS1fIW6:1417088721)*

**Comment by selurvedu** [comment]
:lgtm:

За faker – жирний :heavy_plus_sign:.

---

Reviewed 1 of 1 files at r1, 1 of 1 files at r2, 5 of 5 files at r3, 5 of 5 files at r4, 1 of 1 files at r5.
Review status: all files reviewed at latest revision, 1 unresolved discussion, all commit checks successful.

---

*[op_robot_tests/tests_files/initial_data.py, line 215 \[r5\]](https://reviewable.io:443/reviews/openprocurement/robot_tests/111#-KDUovl7OEIuhD4Yfxrj:-KDUovl7OEIuhD4Yfxrk:-13347624) ([raw file](https://github.com/openprocurement/robot_tests/blob/f386fb48553dc6f322e645eb16304ea3185ead27/op_robot_tests/tests_files/initial_data.py#L215)):*
Можна скоротити так:
```robotframework
if procurement_method_type in ("negotiation", "negotiation.quick"):
```

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/openprocurement/robot_tests/111#-:-KDUogeVp8XRwH0CqHAa:-426506658)*

**Pull request merged**